src/test/java/com/sublinks/sublinksapi/customemoji/dto/CustomEmojiKeywordTest.java

```diff
@@ -0,0 +1,113 @@
+package com.sublinks.sublinksapi.customemoji.dto;
+
+import org.junit.jupiter.api.Test;
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CustomEmojiKeywordTest {
+
+  @Test
+  public void testEquals() {
```

Comment:
Regarding Test Names - Is there an established convention that has been set for naming? I see a couple different methods implemented so far